driver/tests.py

```diff
@@ -18,13 +19,13 @@
 
 def translate():
     """Create translated task."""
-    cmd = ["./fast-downward.py", "--translate",
+    cmd = [sys.executable, "fast-downward.py", "--translate",
```

Comment:
Can we factor this (sys.executable and "fast-downward.py" ) out into run_driver()?

Reply:
I did, but needed another special case for `test_hard_time_limit`.